database/seeders/SubModuleListSeeder.php

```
@@ -0,0 +1,744 @@
+use Illuminate\Support\Facades\DB;
+
+class SubModuleListSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $sml_json = '
```

PHPMD (2.15.0)

The variable $sml_json is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)